core/grpc/src/identifiers.rs

```diff
@@ -46,6 +46,10 @@ pub struct KeyId([u8; ID_LENGTH]);
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, Copy)]
 pub struct RequestId([u8; ID_LENGTH]); // TODO(#2748) rename to InternalRequestId
 
+/// EpochId represents a unique identifier for an epoch/PRSS.
```

Comment:
Do we actually have an advantage in using separate types when everything is based on a `RequestId`?

Reply:
it prevents us from using the wrong request ID by accident, just like how in tfhe-rs all the integer types like lwe dimension, glwe dimension, polynomial size, etc all have different types. they all start with an integer and then immediately converted to the right type.

it would be better if we start with different types in the grpc, but that's a breaking change

Reply:
Ok, makes sense. Can you just add a small comment about this at the definition so we remember the motivation (in particular since it is not reflected in grpc).

Reply:
added module level comment here c980e45b3260f4ea140d6d23fe38167a4758cd36